public/data/shivam-sharma7.json

```
@@ -1,7 +1,7 @@
 {
   "name": "Shivam Sharma",
   "type": "personal",
-  "bio": "Member of @EddieHubCommunity, @fossasia | Full Stack Web Developer | Open Source & Cloud & DevOps Enthusiast |",
+  "bio": "Member of @EddieHubCommunity | Full Stack Web Developer | Open Source & Cloud & DevOps Enthusiast |",
```

```suggestion
  "bio": "Member of @EddieHubCommunity | Full Stack Web Developer | Open Source & Cloud & DevOps Enthusiast.",
```
Replace the vertical bar(`|`) with a full stop(`.`)

---

Okay @vikasganiga05